Automattic/vip-go-mu-plugins
# Removing old deprecated functions

## Description

This PR removes some of the functions in `vip-deprecated.php`. They are functions outside a namespace that are not being used anymore. We have only removed functions that were marked as deprecated in 2015 and are pre VIP Go related.

## Changelog Description

### Removing deprecated functions

We have removed the following deprecated functions:

- `wpcom_vip_load_helper`
- `wpcom_vip_load_helper_wpcom`
- `wpcom_vip_load_helper_stats`
- `vip_allow_title_orphans`
- `wpcom_vip_disable_enhanced_feeds`
- `vip_remove_enhanced_feed_images`
- `wpcom_vip_allow_full_size_images_for_real`
- `wpcom_vip_remove_polldaddy_rating`
- `wpcom_vip_disable_postpost`
- `wpcom_disable_mobile_app_promotion`
- `wpcom_vip_enable_term_order_functionality`
- `wpcom_vip_allow_more_html_in_comments`
- `wpcom_vip_disable_devicepx_js`
- `wpcom_vip_disable_hovercards`
- `wpcom_vip_disable_global_terms`
- `wpcom_vip_disable_custom_customizer`
- `wpcom_vip_remove_opensearch`
- `wpcom_uncached_get_post_by_meta`
- `vary_cache_on_function`
- `vip_multiple_moderators`
- `wpcom_vip_disable_zemanta_for_all_users`
- `get_blog_lang_code`
- `wpcom_vip_stats_roles`
- `wpcom_vip_get_most_shared_posts`
- `wpcom_print_sitemap`
- `wpcom_print_news_sitemap`
- `do_sitemap_pings`
- `sitemap_discovery`
- `sitemap_handle_update`
- `wpcom_vip_remove_feed_tracking_bug`
- `wpcom_vip_plugins_ui_disable_activation`
- `wpcom_vip_load_custom_cdn`

Not a lot of significant changes in this patch release, just bugfixes and compatibility improvements.
-->
## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Run mu-plugins normally. You shouldn't be using any of the functions above.
